hooks/open-telemetry/src/main/java/dev/openfeature/contrib/hooks/otel/OpenTelemetryMetricHook.java

```diff
@@ -0,0 +1,167 @@
+package dev.openfeature.contrib.hooks.otel;
+
+import dev.openfeature.sdk.EvaluationContext;
+import dev.openfeature.sdk.FlagEvaluationDetails;
+import dev.openfeature.sdk.FlagMetadata;
+import dev.openfeature.sdk.Hook;
+import dev.openfeature.sdk.HookContext;
+import io.opentelemetry.api.OpenTelemetry;
+import io.opentelemetry.api.common.Attributes;
+import io.opentelemetry.api.common.AttributesBuilder;
+import io.opentelemetry.api.metrics.LongCounter;
+import io.opentelemetry.api.metrics.LongUpDownCounter;
+import io.opentelemetry.api.metrics.Meter;
+import lombok.extern.slf4j.Slf4j;
+
+import java.util.Collections;
+import java.util.List;
+import java.util.Map;
+import java.util.Optional;
+
+import static dev.openfeature.contrib.hooks.otel.OtelCommons.ERROR_KEY;
+import static dev.openfeature.contrib.hooks.otel.OtelCommons.REASON_KEY;
+import static dev.openfeature.contrib.hooks.otel.OtelCommons.flagKeyAttributeKey;
+import static dev.openfeature.contrib.hooks.otel.OtelCommons.providerNameAttributeKey;
+import static dev.openfeature.contrib.hooks.otel.OtelCommons.variantAttributeKey;
+
+
+/**
+ * OpenTelemetry metric hook records metrics at different {@link Hook} stages.
+ */
+@Slf4j
+@SuppressWarnings("PMD.TooManyStaticImports")
+public class OpenTelemetryMetricHook implements Hook {
+
+    private static final String METER_NAME = "go.openfeature.dev";
```

Comment:
I'm a bit confused about this name. Why "go"?

Reply:
arghh, this comes from go sdk. I copied the cosntants and this came with that 😞 . will fix